medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,257 @@
+        """
+        def process_column_header(td):
+            ret = u''
+            if td.a and td.a.img:
+                ret = td.a.img.get('title', td.a.get_text(strip=True))
+            if not ret:
+                ret = td.get_text(strip=True)
+            return ret
+
+        items = []
+        has_next_page = False
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find('table', id='torrent_table')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # ignore next page in RSS mode
+            has_next_page = mode != 'RSS' and html.find('a', class_='pager_next') is not None
+            log.debug(u'More Pages? {0}'.format(has_next_page))
+
+            # Continue only if at least one Release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return {'has_next_page': has_next_page, 'items': []}
+
+            # '', '', 'Name /Year', 'Files', 'Time', 'Size', 'Snatches', 'Seeders', 'Leechers'
+            labels = [process_column_header(label) for label in torrent_rows[0]('td')]
+            group_title = u''
+
+            # Skip column headers
+            for result in torrent_rows[1:]:
+                cells = result('td')
+                result_class = result.get("class")
+                group_index = -2 if "group_torrent" in result_class else 0
+                # if len(cells) < len(labels):
+                #     continue
+                try:
+                    title = result.select('a[href^="torrents.php?id="]')[0].get_text()
+                    title = re.sub('\s+', ' ', title).strip()  # clean empty lines and multiple spaces
+
+                    if "group" in result_class or "torrent" in result_class:
+                        # get international title if available
+                        title = re.sub('.* \[(.*?)\](.*)', r'\1\2', title)
+
+                    if "group" in result_class:
+                        group_title = title
+                        continue
+
+                    for serie in self.absolute_numbering:
+                        if serie in title:
+                            # remove season from title when its in absolute format
+                            title = re.sub('S\d{2}(E\d{2,4})', r'\1', title)
+                            break
+
+                    download_url = urljoin(self.url, result.select('a[href^="torrents.php?action=download"]')[0]['href'])
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(cells[labels.index('Seeders')+group_index].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('Leechers')+group_index].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < self.minseed or leechers < self.minleech:
```

Comment:
See AlphaRatio on how this should look like.

Reply:
What exacly?
That's almost identical to AlphaRatio, the difference is that it's addapted to support Grouping Torrents settings, alpharatio does not support that...

When it's enabled, the layout is a bit different, also the columns, that's why i need the group_index that fix it based on if grouping torrents is enabled or not